test/skimNtuple_HHbtag.cpp

```diff
@@ -2850,7 +2950,8 @@ int main (int argc, char** argv)
 		  except_VsJet = idSF_leg1 * idSF_leg2_deep_vsEle    * idSF_leg2_deep_vsMu;
 		  except_vsMu  = idSF_leg1 * idSF_leg2_deep_vsJet_2d * idSF_leg2_deep_vsEle;
 		  except_vsEle = idSF_leg1 * idSF_leg2_deep_vsJet_2d * idSF_leg2_deep_vsMu;
-		
+		  float except_leg1 = idSF_leg2_deep_vsJet_2d * idSF_leg2_deep_vsEle * idSF_leg2_deep_vsMu;
```

Comment:
declare outside the condition to keep the symmetry with ```float except_VsJet, except_vsMu, except_vsEle;``` (or the other way around, just be consistent)